larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -1883,21 +1885,25 @@ public String doActionBetweenKeys(String key, String value, Properties propertie
 				debugMessage(key + " between key1 '" + key1 + "' and key2 '" + key2 + "'");
 				result = action.format(result, k -> properties.getProperty(keyPrefix + k), key1, key2);
 				i++;
+			} else if (key1 != null || key2 != null) {
+				throw new IllegalArgumentException("Error in Larva scenario file: key1 or key2 of [" + key + i + "] is missing: key1=[" + key1 + "], key2=[" + key2 + "]");
```

Comment:
Ik denk dat dit nog duidelijker is:

```suggestion
				throw new IllegalArgumentException("Error in Larva scenario file: [" + key + i + "] is missing: key1=[" + key1 + "], key2=[" + key2 + "]");
```

Reply:
Ik twijfel. De `keyN` mist niet, maar is incompleet.

Reply:
Ik heb nu een iets andere message gecommit.